.github/workflows/publish.yml

```diff
@@ -28,10 +28,7 @@ jobs:
           tags: |
             type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
             type=ref,event=branch
-            type=ref,event=pr
-            type=semver,pattern={{version}}
```

Comment:
Can we please fix annotation checks too?

Reply:
Sorry which ones? 

> Unchanged files with check annotations 

These? If so I think separate PR using go 1.16+ stdlib..

Reply:
I see some files under the heading "Unchanged files with check annotations".

Reply:
Righto, yes those are unrelated to this PR and started showing as warnings (not CI failure) when we added `go vet` checker.
See: https://github.com/salesforce/sloop/blob/1ac931c9e226ddaa88a393314aeb7fa44219cc59/.github/workflows/test.yml#L35-L38

We chose not to enforce a clean `go vet` until someone got around to fixing the issues raised.

It would be simplest to fix the warnings but I was thinking it might be possible to drop the Afero dependency for the standard library [embed](https://pkg.go.dev/embed) package. 
Maybe I do this as a separate PR when have time (probably soon :sweat_smile:)?

Reply:
Sounds good. Lets fix them in a separate PR.